model/riscv_vmem_pte.sail

```diff
@@ -128,7 +128,7 @@ function update_PTE_Bits(sv_params : SV_Params,
     let pte_flags = [pte_flags with
                       A = 0b1,
                       D = (if update_d then 0b1 else pte_flags[D])];
-    Some(pte[63 .. 8] @ pte_flags.bits())
+    Some(pte[63 .. 8] @ pte_flags[bits])
```

Comment:
I intended `pte_flags.bits` as the syntax to access the underlying bitvector, but I see why this works as `bits` otherwise works like a field representing all the bits in a bitfield.

Reply:
For this PR I think we should use `pte_flags.bits`, as it's the documented syntax and what the model does elsewhere.

Reply:
Oops. I think I just saw the error message about the old bitfield syntax and assumed it was the same issue as the other ones. Thanks for catching that. Updated.